### PR TITLE
standardizing winevent internal logs

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
working on converting winevent over to the standard logging structure

Had to add a new relay type on the `ingest/log` package.

closes https://github.com/gravwell/gravwell/issues/428

### CODE REVIEW

File: ingesters/winevents/main.go [hunk 1]
```diff
@@ -74,27 +72,34 @@ func main() {
...
 	}
-	if !inter {
+	if inter {
+		lg = log.New(os.Stdout)
+	} else {
+		lg = log.NewDiscardLogger()
 		e, err := eventlog.Open(serviceName)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to get event log handle: %v\n", err)
+			lg.Error("failed to get event log handle", log.KVErr(err))
```
Comment (john-floren-gravwell) [inline ingesters/winevents/main.go:81]: We've just set lg to a NewDiscardLogger, so this message is guaranteed to be thrown away every time, right? I think printing to stderr is a better idea.

File: ingesters/winevents/main.go [hunk 2]
```diff
@@ -169,46 +174,27 @@ func debugout(format string, args ...interface{}) {
...
 
-func warnout(format string, args ...interface{}) {
-	if warnW != nil {
-		warnW(format, args...)
+// levelLogger implements the log.LevelRelay interface
+func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg []byte) error {
+	switch lvl {
+	case log.DEBUG:
+		if verbose {
+			fmt.Fprintln(os.Stdout, string(msg))
```
Comment (john-floren-gravwell) [inline ingesters/winevents/main.go:186]: Stdout or Stderr?

Comment (kris-watts-gravwell) [reply ingesters/winevents/main.go:186]: i think for pure debug it should be stdout.  I think the O.G. choice of stderr was incorrect

File: ingesters/winevents/main.go (hunk 1 above)
Comment (kris-watts-gravwell) [reply ingesters/winevents/main.go:81]: agreed, fix incoming

**Review by john-floren-gravwell — APPROVED** [review]

**Pull request merged**